src/screens/SDKPayment/SDKPaymentUtils.res

```diff
@@ -43,6 +43,7 @@ let initialValueForForm: HSwitchSettingTypes.profileEntity => SDKPaymentTypes.pa
         number: "1234567890",
         country_code: "+1",
       },
+      email: "billing_email@gmail.com",
```

Comment:
please re-confirm the email value

Reply:
Check the order also I guess it should be before Billing & Shipping Address